static/js/components/tiles/chart_footer.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Footer for charts in tiles.
+ */
+
+import _ from "lodash";
+import React from "react";
+
+import { NL_SOURCE_REPLACEMENTS } from "../../constants/app/nl_interface_constants";
+import { urlToDomain } from "../../shared/util";
+import { isNlInterface } from "../../utils/nl_interface_utils";
+
+interface ChartFooterPropType {
+  sources: Set<string>;
```

Comment:
clarify with a comment here that these should be full urls?

Reply:
done